Doc/library/urllib.parse.rst

```diff
@@ -312,6 +312,10 @@ or on combining URL components into a URL string.
    ``#``, ``@``, or ``:`` will raise a :exc:`ValueError`. If the URL is
    decomposed before parsing, no error will be raised.
 
+   Following the specification in WHATWG which updates RFC 3986, ASCII newline
+   ``\n``, ``\r`` or ``\r\n`` and tab ``\t`` characters are stripped from
```

Comment:
no need to mention \r\n here, that could mislead the reader into thinking it was special (and that alternate combinations were not).  just listing \n, \r, and \t is sufficient.

Reply:
Oh, okay! I meant to expand the meaning of ASCII newline (which can include `\r\n`). But I get the reasoning and will remove it.